apps/gpc-verifier/app/hooks/useProofRequest.ts

```diff
@@ -0,0 +1,147 @@
+import { useMemo } from "react";
+import {
+  serializeGPCProofConfig,
+  gpcBindConfig,
+  GPCProofConfig,
+  PODMembershipLists
+} from "@pcd/gpc";
+import { POD_INT_MAX } from "@pcd/pod";
+import { generateSnarkMessageHash } from "@pcd/util";
+
+// https://docs.pcd.team/types/_pcd_gpc.GPCProofConfig.html
+const proofConfig: GPCProofConfig = {
+  // Each POD object mentioned here must be provided in GPCProofInputs.
+  // The names assigned here are used to refer these PODs in GPCProofInputs,
+  // as well as elsewhere in configuration.
+  pods: {
+    govID: {
+      entries: {
+        // Prove the presence of an entry called "idNumber", hide its value,
+        // and also prove that it is not a member of the sanctionsList.
+        idNumber: { isRevealed: false, isNotMemberOf: "sanctionsList" },
+        // Prove the presence of an entry called "firstName", hide its value.
+        firstName: { isRevealed: false },
+        // Prove the presence of an entry called "lastName", hide its value.
+        lastName: { isRevealed: false },
+        // Prove the presence of an entry called "age", hide its value.
```

Comment:
This is a bit unrealistic because it means your ID needs to be re-issued when your age changes.  Also if you were trying to prove your age was <18 (e.g. to qualify for child services) you'd need PODs to expire.

While somewhat less readable, it might be better to make this a date of birth encoded as an integer, and add some logic here to fill in the lower-bound based on today's date.  If not we should at least put a comment explaining that we realize is an unrealistic demo for the reasons above, though my take is that this feature is core enough to the concept of this demo that we should do it closer to right.

Reply:
yeah, maybe have a date of birth encoded as an integer is the best option here; if we want to hide this value, and use inRange to check it.